host/app/components/preview.gts

```diff
@@ -109,10 +109,20 @@ export default class Preview extends Component<Signature> {
       this,
       () => {
         if (this.args.card.type === 'new') {
+          if (this.args.card.initialCardResource) {
+            if (!("attributes" in this.args.card.initialCardResource)) {
+              throw new Error('initialCardResource must have attributes');
+            }
+            if (!("meta" in this.args.card.initialCardResource)) {
+              throw new Error('initialCardResource must have meta');
+            }
+            if (this.args.card.initialCardResource.meta.adoptsFrom !== this.args.card.cardSource) {
+              throw new Error('initialCardResource must have meta.adoptsFrom that matches the cardSource');
+            }
```

Comment:
we should be able avoid these checks because the type for `initialCardResource` is a `LooseCardResource`. are you encountering TS issues around these fields without these checks? (incidentally it's ok to have no attributes--that's a valid resource)

Reply:
Yeah, I realized that it's ok. Will remove